frontend/src/metabase/admin/settings/components/EmbeddingSettings/EmbeddingSecuritySettings/CorsInputWidget.tsx

```diff
@@ -25,7 +25,7 @@ export const CorsInputWidget = () => {
       description={
         <Group align="center" gap="sm">
           <Text c="text-medium" fz="md">
-            {jt`Add the website domains where you want to allow Embedded Analytics JS and SDK. SDK using API keys can only run on localhost.. ${(
+            {jt`Add the website domains where you want to allow Modular embedding and Modular embedding SDK. SDK using API keys can only run on localhost.. ${(
```

Comment:
```suggestion
            {jt`Add the website domains where you want to allow modular embedding (with or without SDK). API keys can only be used on localhost.. ${(
```

This could use some eyes @jeff-bruemmer